src/Modules/OrchardCore.Commerce.Payment.Stripe/Drivers/StripeApiSettingsDisplayDriver.cs

```diff
@@ -52,22 +57,40 @@ public override async Task<IDisplayResult> EditAsync(ISite model, StripeApiSetti
 
         context.AddTenantReloadWarningWrapper();
 
-        return Initialize<StripeApiSettingsViewModel>("StripeApiSettings_Edit", model =>
-            {
-                model.PublishableKey = section.PublishableKey;
+        return Initialize<StripeApiSettingsViewModel>(
+                "StripeApiSettings_Edit",
+                model =>
+                {
+                    model.PublishableKey = section.PublishableKey;
+
+                    model.AccountId = section.AccountId;
 
-                // Decrypting key.
-                model.SecretKey = section.DecryptSecretKey(_dataProtectionProvider, _logger);
+                    // Decrypting key.
+                    model.SecretKey = section.DecryptSecretKey(_dataProtectionProvider, _logger);
 
-                model.WebhookSigningSecret = section.WebhookSigningSecret.DecryptStripeApiKey(_dataProtectionProvider, _logger);
-            })
+                    model.WebhookSigningSecret = section.WebhookSigningSecret.DecryptStripeApiKey(
+                        _dataProtectionProvider,
+                        _logger
+                    );
+                }
+            )
             .PlaceInContent()
             .OnGroup(SettingsGroupId);
     }
 
-    public override async Task<IDisplayResult> UpdateAsync(ISite model, StripeApiSettings section, UpdateEditorContext context)
+    public override async Task<IDisplayResult> UpdateAsync(
+        ISite model,
+        StripeApiSettings section,
+        UpdateEditorContext context
+    )
     {
-        if (await context.CreateModelMaybeAsync<StripeApiSettingsViewModel>(Prefix, AuthorizeAsync) is { } viewModel)
+        if (
+            await context.CreateModelMaybeAsync<StripeApiSettingsViewModel>(
+                Prefix,
+                AuthorizeAsync
+            ) is
+            { } viewModel
```

Comment:
What is the purpose of this strange formatting? I really don't like the line break after the `is` operator.

Reply:
CSharpier

Reply:
I have a PR about it in main repo. Unpleasant is pretty subjective. I think VS doesn'nt reformat these because it is proper formating but I will comply to your request when I get some time. Thanks for your review.